content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms.md

```diff
@@ -40,7 +40,7 @@ You can set the following top-level keys for each issue form.
 | `description` | A description for the issue form template, which appears in the template chooser interface. | Required | String |
 | `body` | Definition of the input types in the form. | Required | Array |
 | `assignees` | People who will be automatically assigned to issues created with this template. | Optional | Array or comma-delimited string |
-| `labels` | Labels that will automatically be added to issues created with this template. | Optional | String |
+| `labels` | Labels that will automatically be added to issues created with this template. | Optional | Array |
```

Comment:
👋🏻 @EmilySeville7cfg, we describe the type for `assignees` in the line above as "Array or comma-delimited string", and I believe `labels` behaves the same way. Let's use that some description in this line, too.

```suggestion
| `labels` | Labels that will automatically be added to issues created with this template. | Optional | Array or comma-delimited string |
```